charts/nginz/values.yaml

```diff
@@ -363,6 +363,11 @@ nginx_conf:
       - staging
       disable_zauth: true
       basic_auth: true
+    - path: ~* /i/teams/([^/]*)/features/([^/]*)/(un)?locked$
+      envs:
+      - staging
+      disable_zauth: true
+      basic_auth: true
```

Comment:
```suggestion
```

The `basic_auth` hack only works on staging (hegemony).  It's not used in the k8s environment.  (There was no way for you to know this, but it may actually be pretty bad to add this here, since the `basic_auth` will (probably?) be ignored, and that means that the end-point will be publicly accessible.)